src/lib/utils.js

```diff
@@ -3,18 +3,19 @@
 const crypto = require('crypto')
 
 /**
- * Get all possible pair combinations from an array.
+ * Get all possible pair combinations from an array, order sensitive.
```

Comment:
you can still manually specify pairs. This just makes the default behavior actually do what it claims.